libr/debug/debug.c

```diff
@@ -95,8 +97,13 @@ static int r_debug_bp_hit(RDebug *dbg, RRegItem *pc_ri, ut64 pc, RBreakpointItem
 	dbg->reason.bp_addr = b->addr;
 
 	/* inform the user of what happened */
-	eprintf ("hit %spoint at: %"PFMT64x "\n",
-			b->trace ? "trace" : "break", pc);
+	if (dbg->corebind.config) {
+		showinfo = r_config_get_i (dbg->corebind.config, "cmd.hitinfo");
```

Comment:
just make this hitinfo variable live inside RDebug struct and add a `set` callback to the rconfig key to update the RDebug field. this way you can remove the rconfig dependency and will run fast because checking a field is faster than doing the same with a hashtable